apex-sysinfo/build.rs

```diff
@@ -0,0 +1,35 @@
+extern crate bindgen;
+
+use std::env;
+use std::path::PathBuf;
+
+fn main() {
+    // Tell cargo to look for shared libraries in the specified directory
+    println!("cargo:rustc-link-search=/usr/lib");
```

Comment:
What is the advantage of doing this ourselves instead of using e.g. https://docs.rs/sysinfo/latest/sysinfo/